test/integration/fact_check_email_test.rb

```diff
@@ -1,6 +1,15 @@
 require 'integration_test_helper'
 
 class FactCheckEmailTest < ActionDispatch::IntegrationTest
+  setup do
```

Comment:
Some other apps use [Climate Control](https://github.com/thoughtbot/climate_control) to change environment variable in tests which is quite nice, might be worth looking into here, but again realise it's the same behaviour as before, so I don't mind.